util/bump_version.sh

```diff
@@ -43,7 +43,12 @@ read_versions() {
 # in: $1
 read_proto_ver() {
 	local ref=$1
-	git show "$ref":src/network/networkprotocol.h | grep -oE 'LATEST_PROTOCOL_VERSION [0-9]+' | tr -dC 0-9
+	local output=$(git show "$ref":src/network/networkprotocol.cpp 2>/dev/null | tee)
+	if [ -z "$output" ]; then
+		# Fallback to previous file (for tags < 5.10.0)
+		output=$(git show "$ref":src/network/networkprotocol.h)
+	fi
+	echo "$output" | grep -oE 'LATEST_PROTOCOL_VERSION\s+=?\s*[0-9]+' | tr -dC 0-9
```

Comment:
very minor simplification
```suggestion
	local output=$(git show "$ref":src/network/networkprotocol.cpp 2>/dev/null)
	if [ -z "$output" ]; then
		# Fallback to previous file (for tags < 5.10.0)
		output=$(git show "$ref":src/network/networkprotocol.h)
	fi
	grep -oE 'LATEST_PROTOCOL_VERSION\s+=?\s*[0-9]+' <<<"$output" | tr -dC 0-9
```